### PR TITLE
[feat] Add the run name to the batch delete and the batch archive modals

### PR DESCRIPTION
**Pull request by VkoHov**
- Add the run name to the batch delete modal
- Replace `Table` component with `DataList`
- Tune styles

### CODE REVIEW

**Comment by VkoHov** [comment]
Resolves [#1593](https://github.com/aimhubio/aim/issues/1593)

**Review by Hamik25 — APPROVED** [review]

File: aim/web/ui/src/components/Table/Table.scss [hunk 1]
```diff
@@ -98,23 +98,30 @@
...
+    position: relative;
     &.isDisabled {
       justify-content: flex-start;
     }
     &__deleteButton {
       display: none;
+      position: absolute;
+      top: 50%;
+      right: -1rem;
+      transform: translate(-50%, -50%);
```
Comment (KaroMourad) [inline aim/web/ui/src/components/Table/Table.scss:110]: let's replace `right: -1rem` with `right: 0` and `transform: translate(-50%, -50%);` 
with `transform: translateY(-50%);`

File: aim/web/ui/src/components/Table/Table.scss [hunk 2]
```diff
@@ -98,23 +98,30 @@
...
     }
     &__date {
+      padding-right: 2.875rem;
```
Comment (KaroMourad) [inline aim/web/ui/src/components/Table/Table.scss:113]: after the above-mentioned changes we can decrease  `padding-right: 2.875rem;`

Comment (KaroMourad) [inline aim/web/ui/src/components/Table/Table.scss:113]: after the above-mentioned changes we can decrease  `padding-right: 2.875rem;`

File: aim/web/ui/src/pages/RunDetail/RunDetail.scss [hunk 3]
```diff
@@ -370,13 +380,13 @@
...
-            height: 3.5rem;
-            padding: 0.625rem 0.5rem;
+            height: auto;
+            padding: 0.5rem 0.5rem;
             display: flex;
             align-items: flex-start;
             cursor: pointer;
             text-decoration: unset;
-            margin-bottom: 0.375rem;
+            margin-bottom: 0.125rem;
```
Comment (KaroMourad) [inline aim/web/ui/src/pages/RunDetail/RunDetail.scss:389]: let's use space units if available  e.g. ` 0.125rem`

Comment (KaroMourad) [inline aim/web/ui/src/pages/RunDetail/RunDetail.scss:389]: let's use space units if available  e.g. ` 0.125rem`

**Review by KaroMourad — APPROVED** [review]
lgtm

**Pull request merged**